src/main/java/duke/command/DoneCommand.java

```diff
@@ -0,0 +1,46 @@
+package duke.command;
+
+import duke.TaskList;
+/**
+ * Represents a command that marks tasks on the task list as done.
+ */
+public class DoneCommand implements Command {
+
+    private final int id;
```

Comment:
I believe final variables should be in all capital letters, including an underscore if there are spaces present.

Reply:
Thanks for your suggestion, but I believe you have made a mistake. Only static final (constants) are in screaming snake case, or what you refer to as in all capital letters.